src/coreclr/jit/morphblock.cpp

```diff
@@ -242,8 +242,6 @@ void MorphInitBlockHelper::PropagateBlockAssertions()
 //
 void MorphInitBlockHelper::PropagateExpansionAssertions()
 {
-    // Consider doing this for FieldByField as well
-    //
```

Comment:
We already do this for the individual fields in that case. And doing it for the base local is unlikely to help anything since the local should be morphed to its fields.